gui/src/2D/advancedDynamicTexture.ts

```diff
@@ -593,11 +610,13 @@ export class AdvancedDynamicTexture extends DynamicTexture {
             if (scene!.isPointerCaptured((<PointerEvent>(pi.event)).pointerId)) {
                 return;
             }
+            //check for focused control and call the onClipboardPointerEvents
+            (this._focusedControl) ? ((pi.type === PointerEventTypes.POINTERDOUBLETAP) ? this._focusedControl.processKeyboard(pi.event) : null) : null;

```

Comment:
I'm not a big fan of multiple ternary operators: it makes things hard to read

Reply:
I thought they are faster than if-else, thats why..